### PR TITLE
Add more details about what this fork is adding

### PR DESCRIPTION
**Pull request by athieriot**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,6 +71,18 @@ Features
 * Programmatic access to Datastore Statistics 
 * Ability to work with Local Datastore (Datastore Emulator) during development phase   
 
+FINkit fork additions
+---------------------
+
+* Support for Generic entities
+    * Most methods working with instances now accept an extra Type parameters
```
Comment (sinsir) [inline README.md]: parameter

File: README.md [hunk 2]
```diff
@@ -71,6 +71,18 @@ Features
...
+    * Methods that used to take a Class as arguement are now taking a more general Type instead
```
Comment (sinsir) [inline README.md]: argument

Comment (sinsir) [reply README.md]: `now accept` instead of `are now taking`

File: README.md [hunk 3]
```diff
@@ -71,6 +71,18 @@ Features
...
+    * This allow to give the Framework more details about Generic types when working with Generic Entities
```
Comment (sinsir) [inline README.md]: 'This provides the framework more details' instead of

'This allow to give the Framework`

**Review by edwardmlyte — APPROVED** [review]

**Pull request merged**